flyway/scxa/migrations/V18__scxa-create-exp-design.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE exp_design
+(
+    accession VARCHAR(255)
```

Comment:
If you replicate columns between tables in this way, then joins will be complex (you'll have to join on the `exp accession`, `annot_type/ column type` and `annot/ column name`). Plus it's error-prone to have the same info in two places. 

What I meant by my earlier suggestion was that you could remove the duplicated fields from this table entirely, and replace them with a single integer FK pointing to the other table. exp_design with then have only `exp_column_id` (the foreign key), `annot_value` and `annot_ont_uri`.

The PK in the other table is then just an integer auto-incremented SERIAL.

This requires more steps in loading, but it's efficient in reducing the amount of replication across rows in the experiment design table, and joins are simplified.

This is analogous with what I'm proposing in https://github.com/ebi-gene-expression-group/atlas-schemas/pull/27.